packages/webview_flutter/webview_flutter_android/lib/src/v4/src/android_webview_controller.dart

```diff
@@ -258,6 +266,15 @@ class AndroidWebViewController extends PlatformWebViewController {
       _androidWebViewParams.androidWebViewProxy
           .setWebContentsDebuggingEnabled(enabled);
 
+  /// Enables or disables file access within WebView.
+  ///
+  /// This enables or disables file system access only. Assets and resources are
+  /// still accessible using file:///android_asset and file:///android_res. The
+  /// default value is true for apps targeting Build.VERSION_CODES.Q and below,
+  /// and false when targeting Build.VERSION_CODES.R and above.
+  Future<void> enableFileAccess(bool enabled) =>
+      _webView.settings.setAllowFileAccess(enabled);
```

Comment:
Ah, I forgot about this one. This is already called in the `loadFile` method. `enableFileAccess` can be removed.